router/worker.go

```diff
@@ -409,6 +409,10 @@ func (w *worker) processDestinationJobs() {
 					result, err := getIterableStruct(destinationJob.Message, transformAt)
 					if err != nil {
 						errorAt = routerutils.ERROR_AT_TF
+						jobIds := lo.Map(destinationJob.JobMetadataArray, func(jobMetadata types.JobMetadataT, _ int) int64 {
+							return jobMetadata.JobID
+						})
+						w.logger.Errorf("transformer response unmarshal error for message: %s, jobs: %v", string(destinationJob.Message), jobIds)
```

Comment:
A bit concerned with printing unbounded large payloads. Would be possible to trim down/limit the size?
Also, can we use structured logging 
```suggestion
						w.logger.Errorw("transformer response unmarshal error for message: %s, jobs: %v", "message", string(destinationJob.Message), "jobIDs", jobIds)
```